docs/index.md

```diff
@@ -0,0 +1,26 @@
+# Index
```

Comment:
Now that I think about it, let's rename the file to `Readme.md` so that it shows up on GitHub automatically when you open the `docs` folder.